### ISSUE TITLE
Rendering Block Entity after a few minutes

### ISSUE THREAD

**Issue by SabaraOne** [simeonradivoev/MatterOverdrive#602]
I'm trying to play a custom modpack that includes MO. A list of mods installed is as follows

- Metroid Cubed 3 0.3.27
- AdminCommandsToolbox 0.0.2a
- AntiIDConflict 1.3.5 (The JAR says latest but MultiMC reads it as 1.3.5 in the metadata)
- CodeChickenCore/NEI 1.0.7.47/1.0.5.120
- DragonAPI/RotaryCraft/Electricraft/ReactorCraft v16d
- CondensedOres v16a (Not actually sure what this one does, but why not?)
- Enhanced Redstone/GeoStrata v16b
- Extended Shaders 1.1
- Matter Overdrive 4.2
- Optifine HD U_D6
- Realistic Terrain Generator 1.1.1
- Waila 1.5.10
- Waila Harvestability 1.1.6
- Waila NBT 1.4

After about two or three minutes of gameplay I get a crash. This is the latest report.
```
---- Minecraft Crash Report ----
// You should try our sister game, Minceraft!

Time: 12/2/16 7:11 PM
Description: Rendering Block Entity

java.lang.IllegalStateException: Already tesselating!
	at net.minecraft.client.renderer.Tessellator.func_78371_b(Tessellator.java:374)
	at net.minecraft.client.renderer.Tessellator.func_78382_b(Tessellator.java:364)
	at matteroverdrive.util.RenderUtils.drawPlaneWithUV(RenderUtils.java:221)
	at matteroverdrive.util.RenderUtils.drawPlane(RenderUtils.java:211)
	at matteroverdrive.util.RenderUtils.drawPlane(RenderUtils.java:203)
	at matteroverdrive.util.RenderUtils.beginDrawinngBlockScreen(RenderUtils.java:481)
	at matteroverdrive.client.render.tileentity.TileEntityRendererHoloSign.func_147500_a(TileEntityRendererHoloSign.java:43)
	at net.minecraft.client.renderer.tileentity.TileEntityRendererDispatcher.func_147549_a(SourceFile:100)
	at net.minecraft.client.renderer.tileentity.TileEntityRendererDispatcher.func_147544_a(SourceFile:92)
	at net.minecraft.client.renderer.RenderGlobal.func_147589_a(RenderGlobal.java:781)
	at net.minecraft.client.renderer.EntityRenderer.func_78471_a(EntityRenderer.java:1599)
	at net.minecraft.client.renderer.EntityRenderer.func_78480_b(EntityRenderer.java:1333)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1001)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:898)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:310)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:395)
	at org.multimc.EntryPoint.listen(EntryPoint.java:170)
	at org.multimc.EntryPoint.main(EntryPoint.java:54)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at net.minecraft.client.renderer.Tessellator.func_78371_b(Tessellator.java:374)
	at net.minecraft.client.renderer.Tessellator.func_78382_b(Tessellator.java:364)
	at matteroverdrive.util.RenderUtils.drawPlaneWithUV(RenderUtils.java:221)
	at matteroverdrive.util.RenderUtils.drawPlane(RenderUtils.java:211)
	at matteroverdrive.util.RenderUtils.drawPlane(RenderUtils.java:203)
	at matteroverdrive.util.RenderUtils.beginDrawinngBlockScreen(RenderUtils.java:481)
	at matteroverdrive.client.render.tileentity.TileEntityRendererHoloSign.func_147500_a(TileEntityRendererHoloSign.java:43)

-- Block Entity Details --
Details:
	Name: holo_sign // matteroverdrive.tile.TileEntityHoloSign
	Block type: ID #576 (tile.holo_sign // matteroverdrive.blocks.BlockHoloSign)
	Block data value: 5 / 0x5 / 0b0101
	Block location: World: (-224,64,213), Chunk: (at 0,4,5 in -14,13; contains blocks -224,0,208 to -209,255,223), Region: (-1,0; contains chunks -32,0 to -1,31, blocks -512,0,0 to -1,255,511)
	Actual block type: ID #576 (tile.holo_sign // matteroverdrive.blocks.BlockHoloSign)
	Actual block data value: 5 / 0x5 / 0b0101
Stacktrace:
	at net.minecraft.client.renderer.tileentity.TileEntityRendererDispatcher.func_147549_a(SourceFile:100)
	at net.minecraft.client.renderer.tileentity.TileEntityRendererDispatcher.func_147544_a(SourceFile:92)
	at net.minecraft.client.renderer.RenderGlobal.func_147589_a(RenderGlobal.java:781)
	at net.minecraft.client.renderer.EntityRenderer.func_78471_a(EntityRenderer.java:1599)

-- Affected level --
Details:
	Level name: MpServer
	All players: 1 total; [EntityClientPlayerMP['SabaraOne'/311, l='MpServer', x=-253.70, y=9.62, z=214.30]]
	Chunk stats: MultiplayerChunkCache: 570, 579
	Level seed: 0
	Level generator: ID 04 - RTG, ver 0. Features enabled: false
	Level generator options: 
	Level spawn location: World: (-39,64,-14), Chunk: (at 9,4,2 in -3,-1; contains blocks -48,0,-16 to -33,255,-1), Region: (-1,-1; contains chunks -32,-32 to -1,-1, blocks -512,0,-512 to -1,255,-1)
	Level time: 175846 game time, 230914 day time
	Level dimension: 0
	Level storage version: 0x00000 - Unknown?
	Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
	Level game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: false
	Forced entities: 106 total; [EntityPig['Pig'/514, l='MpServer', x=-212.63, y=64.00, z=214.66], EntityPig['Pig'/515, l='MpServer', x=-213.31, y=64.00, z=212.53], EntityPig['Pig'/516, l='MpServer', x=-217.53, y=64.00, z=213.81], EntityPig['Pig'/517, l='MpServer', x=-212.72, y=66.00, z=209.22], EntityZombie['Zombie'/518, l='MpServer', x=-214.50, y=50.00, z=230.50], EntitySlime['Slime'/519, l='MpServer', x=-221.22, y=61.00, z=249.78], EntitySlime['Slime'/520, l='MpServer', x=-218.22, y=61.00, z=253.22], EntitySmallEnergyRefill['entity.mc3.smallenergyrefill.name'/521, l='MpServer', x=-200.13, y=63.13, z=213.25], EntityMinecartChest['entity.MinecartChest.name'/522, l='MpServer', x=-204.50, y=39.34, z=243.94], EntitySlime['Slime'/11, l='MpServer', x=-235.78, y=65.00, z=185.22], EntityBat['Bat'/34827, l='MpServer', x=-256.97, y=42.00, z=157.72], EntityPig['Pig'/12, l='MpServer', x=-239.88, y=65.00, z=190.06], EntityMinecartChest['entity.MinecartChest.name'/28, l='MpServer', x=-213.50, y=27.34, z=183.44], EntityZombie['Zombie'/29, l='MpServer', x=-210.40, y=27.35, z=180.51], EntityChicken['Chicken'/30, l='MpServer', x=-210.50, y=27.00, z=180.50], EntityChicken['Chicken'/38, l='MpServer', x=-205.47, y=66.00, z=142.34], EntityMinecartChest['entity.MinecartChest.name'/42, l='MpServer', x=-198.50, y=23.34, z=147.53], EntityChicken['Chicken'/44, l='MpServer', x=-197.47, y=66.00, z=154.59], EntityChicken['Chicken'/46, l='MpServer', x=-195.47, y=65.00, z=166.41], EntityChicken['Chicken'/47, l='MpServer', x=-193.38, y=65.00, z=160.09], EntityBat['Bat'/70451, l='MpServer', x=-260.53, y=39.88, z=166.52], EntityBat['Bat'/52, l='MpServer', x=-198.50, y=24.00, z=184.22], EntityCow['Cow'/53, l='MpServer', x=-194.31, y=65.00, z=191.75], EntityMinecartChest['entity.MinecartChest.name'/54, l='MpServer', x=-188.50, y=22.50, z=149.50], EntityCreeper['Creeper'/56, l='MpServer', x=-185.50, y=22.00, z=159.50], EntityBat['Bat'/313, l='MpServer', x=-228.06, y=20.10, z=204.34], EntityChicken['Chicken'/57, l='MpServer', x=-183.47, y=65.00, z=160.19], EntityPig['Pig'/58, l='MpServer', x=-183.69, y=66.00, z=164.50], EntityPig['Pig'/315, l='MpServer', x=-237.50, y=67.00, z=194.69], EntityBat['Bat'/59, l='MpServer', x=-180.75, y=22.00, z=192.59], EntityPig['Pig'/316, l='MpServer', x=-238.47, y=65.00, z=197.47], EntityPig['Pig'/317, l='MpServer', x=-239.84, y=65.00, z=193.03], EntitySpider['Spider'/63294, l='MpServer', x=-219.97, y=66.00, z=164.94], EntitySkeleton['Skeleton'/115518, l='MpServer', x=-190.50, y=67.00, z=219.50], EntitySkeleton['Skeleton'/63296, l='MpServer', x=-220.50, y=66.00, z=169.50], EntitySpider['Spider'/63297, l='MpServer', x=-219.50, y=66.00, z=167.50], EntityCreeper['Creeper'/31045, l='MpServer', x=-327.50, y=64.00, z=238.50], EntitySkeleton['Skeleton'/51013, l='MpServer', x=-201.50, y=66.00, z=144.50], EntityZombie['Zombie'/31046, l='MpServer', x=-333.50, y=64.00, z=234.50], EntityZombie['Zombie'/31047, l='MpServer', x=-332.50, y=64.00, z=237.50], EntityCreeper['Creeper'/51015, l='MpServer', x=-252.50, y=66.00, z=198.50], EntitySkeleton['Skeleton'/105030, l='MpServer', x=-264.50, y=64.00, z=289.50], EntityZombie['Zombie'/31048, l='MpServer', x=-193.03, y=21.00, z=196.41], EntityCreeper['Creeper'/51016, l='MpServer', x=-258.00, y=65.00, z=199.44], EntityChicken['Chicken'/331, l='MpServer', x=-253.56, y=64.00, z=213.56], EntityChicken['Chicken'/332, l='MpServer', x=-252.59, y=64.00, z=223.44], EntitySpider['Spider'/105039, l='MpServer', x=-187.53, y=22.00, z=144.38], EntityZombie['Zombie'/24154, l='MpServer', x=-181.50, y=65.00, z=213.50], EntityBat['Bat'/107616, l='MpServer', x=-231.53, y=15.00, z=135.31], EntityClientPlayerMP['SabaraOne'/311, l='MpServer', x=-253.70, y=9.62, z=214.30], EntityZombie['Zombie'/108913, l='MpServer', x=-240.50, y=67.00, z=161.50], EntityZombie['Zombie'/108915, l='MpServer', x=-297.61, y=65.85, z=144.31], EntitySkeleton['Skeleton'/109947, l='MpServer', x=-331.03, y=63.00, z=197.72], EntitySkeleton['Skeleton'/109948, l='MpServer', x=-187.50, y=22.00, z=140.50], EntityCreeper['Creeper'/84619, l='MpServer', x=-300.50, y=64.00, z=190.50], EntitySpider['Spider'/84618, l='MpServer', x=-262.88, y=64.00, z=281.06], EntityZombie['Zombie'/84621, l='MpServer', x=-243.50, y=63.00, z=228.50], EntityCreeper['Creeper'/84620, l='MpServer', x=-290.50, y=66.00, z=158.50], EntityZombie['Zombie'/84622, l='MpServer', x=-242.50, y=63.00, z=229.50], EntityBat['Bat'/78480, l='MpServer', x=-253.78, y=41.00, z=157.63], EntityMeleeRougeAndroidMob['�3[1] ED-209'/24211, l='MpServer', x=-263.50, y=64.00, z=235.50], EntitySkeleton['Skeleton'/99225, l='MpServer', x=-300.88, y=63.00, z=160.50], EntitySkeleton['Skeleton'/99224, l='MpServer', x=-292.50, y=66.00, z=158.50], EntityCreeper['Creeper'/99227, l='MpServer', x=-294.50, y=66.00, z=158.50], EntityCreeper['Creeper'/99230, l='MpServer', x=-254.50, y=42.00, z=160.50], EntitySlime['Slime'/77217, l='MpServer', x=-247.50, y=64.00, z=283.50], EntityPig['Pig'/420, l='MpServer', x=-316.47, y=63.00, z=219.19], EntityPig['Pig'/421, l='MpServer', x=-309.09, y=63.00, z=219.34], EntityPig['Pig'/422, l='MpServer', x=-309.94, y=66.00, z=214.44], EntityPig['Pig'/423, l='MpServer', x=-323.69, y=65.00, z=206.50], EntityPig['Pig'/424, l='MpServer', x=-317.50, y=64.00, z=234.50], EntityPig['Pig'/425, l='MpServer', x=-314.41, y=64.00, z=234.78], EntityPig['Pig'/426, l='MpServer', x=-315.97, y=64.00, z=234.50], EntityPig['Pig'/427, l='MpServer', x=-313.09, y=64.00, z=233.19], EntityChicken['Chicken'/433, l='MpServer', x=-290.50, y=66.00, z=150.50], EntityChicken['Chicken'/434, l='MpServer', x=-290.50, y=66.00, z=149.50], EntityZombie['Zombie'/45746, l='MpServer', x=-217.50, y=65.00, z=179.50], EntityChicken['Chicken'/435, l='MpServer', x=-290.78, y=66.00, z=152.53], EntityChicken['Chicken'/436, l='MpServer', x=-289.50, y=67.00, z=149.50], EntityPig['Pig'/437, l='MpServer', x=-291.66, y=65.00, z=284.66], EntityCreeper['Creeper'/45749, l='MpServer', x=-202.50, y=66.00, z=138.50], EntityPig['Pig'/438, l='MpServer', x=-289.50, y=65.00, z=284.50], EntitySkeleton['Skeleton'/45750, l='MpServer', x=-203.84, y=66.00, z=141.06], EntityPig['Pig'/439, l='MpServer', x=-291.97, y=65.00, z=281.81], EntityPig['Pig'/440, l='MpServer', x=-281.03, y=63.00, z=284.66], EntityChicken['Chicken'/471, l='MpServer', x=-273.50, y=65.00, z=184.50], EntityChicken['Chicken'/472, l='MpServer', x=-272.50, y=65.00, z=188.50], EntityChicken['Chicken'/473, l='MpServer', x=-284.44, y=66.00, z=181.56], EntityChicken['Chicken'/474, l='MpServer', x=-276.47, y=65.00, z=194.44], EntityChicken['Chicken'/475, l='MpServer', x=-276.34, y=64.00, z=210.53], EntityMeleeRougeAndroidMob['�3�3[1] Optimus Prime'/480, l='MpServer', x=-261.72, y=54.00, z=162.63], EntityChicken['Chicken'/481, l='MpServer', x=-262.44, y=65.00, z=198.56], EntityChicken['Chicken'/482, l='MpServer', x=-262.50, y=63.00, z=215.50], EntityChicken['Chicken'/490, l='MpServer', x=-241.50, y=66.00, z=140.50], EntityChicken['Chicken'/491, l='MpServer', x=-244.19, y=66.00, z=137.03], EntityChicken['Chicken'/492, l='MpServer', x=-246.50, y=66.00, z=138.50], EntityChicken['Chicken'/494, l='MpServer', x=-241.53, y=65.00, z=190.56], EntityChicken['Chicken'/495, l='MpServer', x=-251.75, y=65.00, z=206.81], EntityChicken['Chicken'/496, l='MpServer', x=-247.56, y=65.00, z=216.59], EntitySkeleton['Skeleton'/26353, l='MpServer', x=-265.50, y=64.00, z=235.50], EntityBat['Bat'/505, l='MpServer', x=-228.44, y=19.10, z=213.63], EntityCreeper['Creeper'/28921, l='MpServer', x=-254.50, y=41.00, z=156.09], EntitySquid['Squid'/506, l='MpServer', x=-229.84, y=61.00, z=240.50], EntityCreeper['Creeper'/28922, l='MpServer', x=-259.50, y=41.00, z=160.50], EntityBat['Bat'/78589, l='MpServer', x=-233.84, y=15.00, z=136.31], EntityBat['Bat'/78588, l='MpServer', x=-258.38, y=42.00, z=157.75]]
	Retry entities: 0 total; []
	Server brand: fml,forge
	Server type: Integrated singleplayer server
Stacktrace:
	at net.minecraft.client.multiplayer.WorldClient.func_72914_a(WorldClient.java:425)
	at net.minecraft.client.Minecraft.func_71396_d(Minecraft.java:2444)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:919)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:310)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:395)
	at org.multimc.EntryPoint.listen(EntryPoint.java:170)
	at org.multimc.EntryPoint.main(EntryPoint.java:54)

-- System Details --
Details:
	Minecraft Version: 1.7.10
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_111, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 1055027832 bytes (1006 MB) / 2275934208 bytes (2170 MB) up to 4772593664 bytes (4551 MB)
	JVM Flags: 3 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xms512m -Xmx5120m
	AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	IntCache: cache: 0, tcache: 0, allocated: 9, tallocated: 61
	FML: MCP v9.05 FML v7.10.99.99 Minecraft Forge 10.13.4.1614 Optifine OptiFine_1.7.10_HD_U_D6 23 mods loaded, 23 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCHIJAAAA	mcp{9.05} [Minecraft Coder Pack] (minecraft.jar) 
	UCHIJAAAA	FML{7.10.99.99} [Forge Mod Loader] (forge-1.7.10-10.13.4.1614-1.7.10-universal.jar) 
	UCHIJAAAA	Forge{10.13.4.1614} [Minecraft Forge] (forge-1.7.10-10.13.4.1614-1.7.10-universal.jar) 
	UCHIJAAAA	CodeChickenCore{1.0.7.47} [CodeChicken Core] (minecraft.jar) 
	UCHIJAAAA	NotEnoughItems{1.0.5.120} [Not Enough Items] (NotEnoughItems-1.7.10-1.0.5.120-universal.jar) 
	UCHIJAAAA	MC3ASMCore{0.1} [Metroid Cubed 3 ASM Core] (minecraft.jar) 
	UCHIJAAAA	extendedshaders{1.1} [Extended Shaders] (minecraft.jar) 
	UCHIJAAAA	<DragonAPI ASM>{0} [DragonAPI ASM Data Initialization] (minecraft.jar) 
	UCHIJAAAA	mc3{0.3.27.0} [Metroid Cubed 3] ([1.7.10]MetroidCubed3V0.3.27.0.jar) 
	UCHIJAAAA	act{0.0.2a_1.7.10} [AdminCommandsToolbox] (AdminCommandsToolbox-0.0.2a_1.7.10.jar) 
	UCHIJAAAA	antiidconflict{1.3.5} [Anti Id Conflict] (AntiIdConflict-LATEST-1.7.10.jar) 
	UCHIJAAAA	Waila{1.5.10} [Waila] (Waila-1.5.10_1.7.10.jar) 
	UCHIJAAAA	DragonAPI{1.0} [DragonAPI] (DragonAPI 1.7.10 V16d.jar) 
	UCHIJAAAA	CondensedOres{1.0} [Condensed Ores] (CondensedOres 1.7.10 V16a.jar) 
	UCHIJAAAA	RotaryCraft{1.0} [RotaryCraft] (RotaryCraft 1.7.10 V16d.jar) 
	UCHIJAAAA	ElectriCraft{1.0} [ElectriCraft] (ElectriCraft 1.7.10 V16d.jar) 
	UCHIJAAAA	ExpandedRedstone{1.0} [Expanded Redstone] (ExpandedRedstone 1.7.10 V16b.jar) 
	UCHIJAAAA	GeoStrata{1.0} [GeoStrata] (GeoStrata 1.7.10 V16b.jar) 
	UCHIJAAAA	mo{0.4.2} [Matter Overdrive] (MatterOverdrive-1.7.10-0.4.2.jar) 
	UCHIJAAAA	ReactorCraft{1.0} [ReactorCraft] (ReactorCraft 1.7.10 V16d.jar) 
	UCHIJAAAA	RTG{1.7.10-1.1.1} [Realistic Terrain Generation] (RTG-1.7.10-1.1.1.jar) 
	UCHIJAAAA	WailaHarvestability{1.1.6} [Waila Harvestability] (WailaHarvestability-mc1.7.10-1.1.6.jar) 
	UCHIJAAAA	WailaNBT{1.4} [Waila NBT] (WailaNBT-1.7.10-1.4.jar) 
	GL info: ' Vendor: 'Intel' Version: '4.3.0 - Build 10.18.15.4248' Renderer: 'Intel(R) HD Graphics 4400'
	Launched Version: MultiMC5
	LWJGL: 2.9.1
	OpenGL: Intel(R) HD Graphics 4400 GL version 4.3.0 - Build 10.18.15.4248, Intel
	GL Caps: Using GL 1.3 multitexturing.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Anisotropic filtering is supported and maximum anisotropy is 16.
Shaders are available because OpenGL 2.1 is supported.

	Is Modded: Definitely; Client brand changed to 'fml,forge'
	Type: Client (map_client.txt)
	Resource Packs: []
	Current Language: English (US)
	Profiler Position: N/A (disabled)
	Vec3 Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	Anisotropic Filtering: Off (1)`
```
I suspect MO of causing the crash simply because I don't recognize any other mod outside of the listing of all active mods, but If it isn't I'd be happy to know who caused the crash.

**Comment by jtrent238** [comment]
It is the Holo Sign from Matter OverDrive

```
        Name: holo_sign // matteroverdrive.tile.TileEntityHoloSign
	Block type: ID #576 (tile.holo_sign // matteroverdrive.blocks.BlockHoloSign)
	Block data value: 5 / 0x5 / 0b0101
	Block location: World: (-224,64,213), Chunk: (at 0,4,5 in -14,13; contains blocks -224,0,208 to -209,255,223), Region: (-1,0; contains chunks -32,0 to -1,31, blocks -512,0,0 to -1,255,511)
	Actual block type: ID #576 (tile.holo_sign // matteroverdrive.blocks.BlockHoloSign)
	Actual block data value: 5 / 0x5 / 0b0101
```

**Comment by SabaraOne** [comment]
Is this a bug or something I should be able to fix? Is there a workaround so I can at least continue playing? I haven't placed a holo sign, heck I haven't even made an isolinear circuit, but knowing what I do about MC worldgen that really doesn't mean much.

**Comment by jtrent238** [comment]
It can be fixed, but it is a bit complicated*. If you send me the save I can fix it for you.

* It requires you to go into NBT Editor and delete the entity, Knowing the exact entity is a real pain, because you might have over 100 entities in a chunk, and they are all listed one by one in NBT Editor.

I have done this before and I have fixed it before, also it was not matter overdrive, but it can still be done that way.

**Comment by SabaraOne** [comment]
Not as hard as you'd think. Since it took a few minutes to crap out I used NBT Explorer to toggle AllowCommands, went Creative, and tpd to the block, breaking it as soon as I came out. Had to toggle KeepInvnetory and kill to get home though. Teh sign was in some sort of MO dungeon, didn't know it even had those. I'm adding Journeymap so I at least have waypoints and a map.
So it was just corrupted NBT  data,right?
